### PR TITLE
next button announcement

### PR DESCRIPTION
**Pull request by lecosson**
Related to : https://oat-sa.atlassian.net/browse/TCA-733
 
- enhance `next` button announcement
- move announcement container to <main>
  
#### How to test

- prepare an instance of TAO
- execute delivery as a test taker
- check that next button announcement contains item number

### CODE REVIEW

File: src/plugins/navigation/next.js [hunk 1]
```diff
@@ -65,14 +65,19 @@ var createElement = function createElement(isLast = false) {
...
  * @param {jQueryElement} $element - the element to update
+ * @param {TestRunner} [testRunner] - the test runner instance
  * @param {Boolean} [isLast=false] - is the current item the last
  */
-const updateElement = function updateElement($element, isLast = false) {
+const updateElement = function updateElement($element, testRunner, isLast = false) {
     const dataType = isLast ? 'end' : 'next';
+    const testContext = testRunner.getTestContext();
+    const nextItemPosition = testContext.itemPosition + 1;
+    if (dataType === 'next') {
```
Comment (atsymuk) [inline src/plugins/navigation/next.js:75]: should not we assign title in case of `dataType ="end"`

**Review by atsymuk — CHANGES_REQUESTED** [review]

**Review by bziondik — APPROVED** [review]
Code review only
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [ ]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

File: src/plugins/navigation/next.js [hunk 2]
```diff
@@ -40,13 +40,13 @@ import buttonTpl from 'taoQtiTest/runner/plugins/templates/button';
 var buttonData = {
     next: {
         control: 'move-forward',
-        title: __('Submit and go to the next item'),
+        title: 'Submit and go to the item %s',
```
Comment (bziondik) [inline src/plugins/navigation/next.js:43]: Why you removed translation `__(` ?

File: src/plugins/navigation/next.js [hunk 3]
```diff
@@ -40,13 +40,13 @@ import buttonTpl from 'taoQtiTest/runner/plugins/templates/button';
...
         icon: 'forward',
         text: __('Next')
     },
     end: {
         control: 'move-end',
-        title: __('Submit and go to the end of the test'),
+        title: 'Submit and go to the end of the test',
```
Comment (bziondik) [inline src/plugins/navigation/next.js:49]: Why you removed translation `__(` ?

File: src/plugins/navigation/next.js [hunk 4]
```diff
@@ -65,14 +65,17 @@ var createElement = function createElement(isLast = false) {
...
  * @param {jQueryElement} $element - the element to update
+ * @param {TestRunner} [testRunner] - the test runner instance
  * @param {Boolean} [isLast=false] - is the current item the last
  */
-const updateElement = function updateElement($element, isLast = false) {
+const updateElement = function updateElement($element, testRunner, isLast = false) {
     const dataType = isLast ? 'end' : 'next';
+    const testContext = testRunner.getTestContext();
+    const nextItemPosition = testContext.itemPosition + 1;
+    $element.attr('title', __(buttonData[dataType].title, nextItemPosition));
```
Comment (bziondik) [inline src/plugins/navigation/next.js:75]: Please, translate it definition of `title`

**Review by bziondik — CHANGES_REQUESTED** [review]

**Review by atsymuk — APPROVED** [review]
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by bziondik — APPROVED** [review]

**Pull request merged**